my-dexit.conf.js

```diff
@@ -0,0 +1,82 @@
+// Karma configuration
```

Comment:
I think it's a good practice to make a file name descriptive ---- 'my-dexit.conf.js'